mappings/net/minecraft/block/AnvilBlock.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_2199 net/minecraft/block/AnvilBlock
-	FIELD field_17349 CONTAINER_NAME Lnet/minecraft/class_2588;
+	FIELD field_17349 SCREEN_TITLE Lnet/minecraft/class_2588;
```

Comment:
IMO this should be `INVENTORY_NAME` since really that's what it is.

Reply:
actually, `INVENTORY_TITLE` because it is a constant, so it isn't necessarily the screen title. The screen title would be either the inventory title constant, or a custom name.

Reply:
```suggestion
	FIELD field_17349 TITLE Lnet/minecraft/class_2588;
```